libethereum/BlockChainSync.cpp

```diff
@@ -137,6 +137,7 @@ BlockChainSync::BlockChainSync(EthereumHost& _host):
 	m_startingBlock(_host.chain().number()),
 	m_lastImportedBlock(m_startingBlock)
 {
+	m_state = SyncState::Idle;
```

Comment:
Why not change the value in the member itself? reassigning it here might be confusing.